### PR TITLE
Add CMakeLists.txt for building with CMake.

### PR DESCRIPTION
**Pull request by jeffdav**
This will give users an alternative to SPM for consuming this library.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    Sources/WinAppDriver/ReexportWebDriver.swift
+    Sources/WinAppDriver/Win32Error.swift
+    Sources/WinAppDriver/Win32ProcessTree.swift
+    Sources/WinAppDriver/WinAppDriver+Attributes.swift
+    Sources/WinAppDriver/WinAppDriver+Capabilities.swift
+    Sources/WinAppDriver/WinAppDriver.swift
+    Sources/WinAppDriver/WindowsSystemPaths.swift)
+  target_link_libraries(WinAppDriver PRIVATE
+    WebDriver)
+endif()
```
Comment (compnerd) [inline CMakeLists.txt:44]: Should we do the CMake recommendation and create sub-CMakeLists.txt for the libraries?

**Review by compnerd — APPROVED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (jeffdav) [reply CMakeLists.txt:44]: Oh, if that's a thing.  Do you have a link?  Or do I just create `Sources/WinAppDriver/CMakeLists` and do an `add_subdirectory()`?

Comment (jeffdav) [reply CMakeLists.txt:35]: How's that look, @compnerd ?

File: Sources/WebDriver/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+add_library(WebDriver
```
[the remark this answers is not in the document]
Comment (jeffdav) [reply Sources/WebDriver/CMakeLists.txt:1]: I was going to let the arc build take care of that.  :)  
But yes, I can.

[the remark this answers is not in the document]
Comment (jeffdav) [reply Sources/WebDriver/CMakeLists.txt:1]: Hmm, getting a strange link error.  Any ideas?

[the remark this answers is not in the document]
Comment (jeffdav) [reply Sources/WebDriver/CMakeLists.txt:1]: (I must be setting up the env wrong.)

**Review by tristanlabelle — APPROVED** [review]

**Pull request merged**